doc/GUIDE.md

```diff
@@ -0,0 +1,822 @@
+# Jextract Guide
+
+The jextract tool parses header (.h) files of native libraries, and generates Java code,
```

Comment:
Aren't .c files also viable for jextract?

Reply:
Technically I don't think the extension matters, but C header files typically have the `.h` extension.